specification/loadtestservice/data-plane/Microsoft.LoadTestService/stable/2022-11-01/loadtestservice.json

```diff
@@ -695,52 +695,12 @@
       }
     },
     "/test-runs/{testRunId}": {
-      "delete": {
-        "tags": [
-          "TestRun"
-        ],
-        "summary": "Delete a test run by its name.",
-        "operationId": "TestRun_Delete",
-        "produces": [
-          "application/json"
-        ],
-        "parameters": [
-          {
-            "$ref": "#/parameters/TestRunIdPathParameter"
-          },
-          {
-            "$ref": "#/parameters/ApiVersion"
-          }
-        ],
-        "responses": {
-          "204": {
-            "description": "The test run is deleted."
-          },
-          "default": {
-            "description": "Load Testing service error response.",
-            "schema": {
-              "$ref": "#/definitions/ErrorResponseBody"
-            },
-            "headers": {
-              "x-ms-error-code": {
-                "description": "The error code for specific error that occurred.",
-                "type": "string"
-              }
-            }
-          }
-        },
-        "x-ms-examples": {
-          "Delete a test run": {
-            "$ref": "./examples/DeleteTestRun.json"
-          }
-        }
-      },
       "patch": {
         "tags": [
           "TestRun"
         ],
         "summary": "Create and start a new test run with the given name.",
-        "operationId": "TestRun_CreateOrUpdate",
+        "operationId": "LoadTestRun_CreateOrUpdate",
```

Comment:
Should it be, LoadTestRun_CreateOrUpdateTestRun?

Reply:
changed